plugins/sinks/wavefront/wavefront.go

```diff
@@ -158,6 +159,25 @@ func (sink *wavefrontSink) ExportData(batch *metrics.DataBatch) {
 	sink.send(batch)
 }
 
+func (wf *wavefrontSink) ExportEvent(event *metrics.Event) {
+	tags := make(map[string]string)
```

Comment:
Is this necessary? Why not just add the global tags to the event.Tags since each event has its own map of tags.